PocketBeagle/Grove/Lesson-1.Control-the-Light/README.md

```diff
@@ -0,0 +1,29 @@
+# Description:
+
+In this lesson, students will light up the RGB LED, and learn how to use Slide Potentiometer and Rotary Angle Sensor to change the light of RGB LED.
+
+## Hardware Requirement:
+
+- [Grove - Slide Potentiometer](https://www.seeedstudio.com/Grove-Slide-Potentiometer.html)
+- [Grove - Rotary Angle Sensor](http://wiki.seeedstudio.com/Grove-Rotary_Angle_Sensor/)
+- [Grove - Chainable RGB LED](http://wiki.seeedstudio.com/Grove-Chainable_RGB_LED/)
+
+
+## Hardware Connection
+ 
+- Plug the Grove - Slide Potentiometer into **A0** port
+- Plug the Grove - Rotary Angle Sensor into **A5** port
+- Grove - Chainable RGB LED into **A2** port
+- Power the Pocket Beagle via the **micro USB** port
+
+![](../img/_DAS6312-1.jpg)
+
+
+## Software
+
+- Step 1. Enter Cloud9 by typing IP of Pocket Beagle
```

Comment:
PocketBeagle

We'll need to have general instructions up-front on finding PocketBeagle's IP address. I'd like a little script that prints it out on the LCD, perhaps configuring it to show on startup. https://www.geeksforgeeks.org/display-hostname-ip-address-python/